deployment/cfn/worker.py

```diff
@@ -240,6 +248,10 @@ def get_cloud_config(self):
         return ['#cloud-config\n',
                 '\n',
                 'write_files:\n',
+                '  - path: /etc/rf.d/env/RF_STACK_COLOR\n',
```

Comment:
Where & when is `RF_STACK_COLOR` created? I don't see it defined in ansible group_vars.